File: c++/Product.cpp
#include <bits/stdc++.h>

using namespace std;

class Product
{
private:
    int price;
    string idProduct;

public:
    Product()
    {
        this->idProduct = "-";
    }

    Product(int price, string idProduct)
    {
        this->price = price;
        this->idProduct = idProduct;
    }

    void setPrice(int price)
    {
        this->price = price;
    }

    void setIdProduct(string idProduct)
    {
        this->idProduct = idProduct;
    }

    int getPrice()
    {
        return this->price;
    }

    string getIdProduct()
    {
        return this->idProduct;
    }

    ~Product(){};
};

File: python/Product.py
class Product():

    def __init__(self, price=0, idProduct="-"):
        self.__price = price
        self.__idProduct = idProduct

    def setPrice(self, price):
        self.__price = price

    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct

    def getPrice(self):
        return self.__price

    def getIdProduct(self):
        return self.__idProduct
